codecarbon/external/hardware.py

```diff
@@ -90,6 +101,28 @@ def _get_power_from_cpus(self) -> Power:
                 power += value
         return Power.from_watts(power)
 
+    def _get_power_from_constant(self) -> Power:
+        """
+        Get CPU power from constant mode
+        :return: power in KW
+        """
+        cpu_info = cpuinfo.get_cpu_info()
+        if cpu_info:
+            model_raw = cpu_info["brand_raw"]
+            model = model_raw.split(" CPU")[0].replace("(R)", "").replace("(TM)", "")
```

Comment:
Let's extract this line into a separate function so we can more easily test it. The reasoning behind that is that CPU-Info has the following warning:
```
These fields are pulled directly from the CPU and are unverified. They may contain expected results. Other times they may contain wildly unexpected results or garbage. So it would be a bad idea to rely on them.
```

I'm guessing that parsing this field is gonna fails in various condition, ARM, AMD, etc... It would be easier to update and test the parsing if the logic is isolated into a separate function